### PR TITLE
Write images with bounding boxes to tensorboard

### PR DESCRIPTION
**Pull request by FrankBrongers**
- added imports to infer.py
- added image logging code to infer.py

Might result in conflicts after merging 'writing AP and AR to tensorboard.':
- changed infer method call
- added logdir arg on infer in main.py

### CODE REVIEW

File: retinanet/main.py [hunk 1]
```diff
@@ -93,7 +94,7 @@ def load_model(args, verbose=False):
 
     elif args.command == 'infer' and ext in ['.engine', '.plan']:
         model = None
-    
+
```
Comment (klausondrag) [inline retinanet/main.py]: Unnecessary whitespace change

File: retinanet/main.py [hunk 2]
```diff
@@ -120,9 +121,9 @@ def worker(rank, args, world, model, state):
 
     if args.command == 'train':
         train.train(model, state, args.images, args.annotations,
-            args.val_images or args.images, args.val_annotations, args.resize, args.max_size, args.jitter, 
-            args.batch, int(args.iters * args.schedule), args.val_iters, not args.full_precision, args.lr, 
-            args.warmup, [int(m * args.schedule) for m in args.milestones], args.gamma, 
+            args.val_images or args.images, args.val_annotations, args.resize, args.max_size, args.jitter,
```
Comment (klausondrag) [inline retinanet/main.py]: Whitespace

**Review by klausondrag — CHANGES_REQUESTED** [review]
Great job overall, almost ready to merge. Please fix the whitespace, then it's good go! (See also this comment on another pull request https://github.com/klausondrag/retinanet-examples/pull/2#pullrequestreview-395903544 )

**Review by FrankBrongers — COMMENTED** [review]
Reset whitespaces
